tests/e2e/monitoring_test.go

```diff
@@ -23,6 +30,177 @@ type MonitoringTestCtx struct {
 	*TestContext
 }
 
+// setupMonitoringCRDs ensures that all required CRDs for monitoring tests are installed.
+func (tc *MonitoringTestCtx) setupMonitoringCRDs() {
+	// Install MonitoringStack CRD
+	if !tc.isMonitoringStackCRDAvailable() {
+		tc.createMonitoringStackCRD()
+	}
+
+	// Install TempoMonolithic CRD
+	if !tc.isTempoMonolithicCRDAvailable() {
+		tc.createTempoMonolithicCRD()
+	}
+
+	// Install TempoStack CRD
+	if !tc.isTempoStackCRDAvailable() {
+		tc.createTempoStackCRD()
+	}
+}
```

Comment:
One more thing, I would expect these CRDs to be installed if you add installation of observability operators, similar how it was done here: https://github.com/opendatahub-io/opendatahub-operator/pull/2014/files#diff-fff639351778722a40b1e9120a83b1a752649ad5668c3886247d24d350a78bbcR54